File: types/question.go
package types

type Question struct {
	Id           *uint   `json:"id,omitempty"`
	SurveyId     *uint   `json:"surveyId,omitempty"`
	QuestionText string  `json:"questionText"`
	Image        *string `json:"image,omitempty"`
	QuestionType *string `json:"questionType,omitempty"`
	MinLabel     string  `json:"minLabel"`
	MaxLabel     string  `json:"maxLabel"`
	DisplayOrder *uint   `json:"displayOrder,omitempty"`
}

File: client/src/types/Question.ts
export interface Question {
    id?: number,
    // surveyId?: number, // no need in client
    questionText: string,
    image: string,
    // questionType: string, // existing only in server/database, it's useless in client (for now)
    minLabel: string,
    maxLabel: string,
    // displayOrder?: number // existing only in server/database, it's useless in client
}